packages/app/src/components/DBRowTable.tsx

```diff
@@ -25,11 +25,7 @@ import {
   ChartConfigWithDateRange,
   SelectList,
 } from '@hyperdx/common-utils/dist/types';
-import {
-    Granularity,
-    splitAndTrimCSV,
-    splitAndTrimWithBracket,
-} from '@hyperdx/common-utils/dist/utils';
+import { splitAndTrimWithBracket } from '@hyperdx/common-utils/dist/utils';
```

Comment:
`Granularity` and `splitAndTrimCSV` is not used at here anymore, remove it since it cause conflict.